adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapter.java

```diff
@@ -733,17 +738,20 @@ protected long messagePayloadSize(final RoutingContext ctx) {
      * @param messageConsumer The message consumer to receive a command. If {@code null}, no handler is added.
      * @param tenantId The tenant that the device belongs to.
      * @param deviceId The identifier of the device.
+     * @param currentSpan The <em>OpenTracing</em> Span used for tracking the processing of the request.
      */
     private void addConnectionCloseHandler(
             final RoutingContext ctx,
             final MessageConsumer messageConsumer,
             final String tenantId,
-            final String deviceId) {
+            final String deviceId,
+            final Span currentSpan) {
 
         if (messageConsumer != null && !ctx.response().closed()) {
             ctx.response().closeHandler(v -> {
                 LOG.debug("device [tenant: {}, device-id: {}] closed connection before response could be sent",
                         tenantId, deviceId);
+                TracingHelper.logError(currentSpan, "device closed connection");
```

Comment:
is this an error? Maybe the device has good reasons to do so ...

Reply:
yes, right. The error, that is probably following this, will be set on the span in the `if (ctx.response().closed()) {` block above.
I've changed it to a normal log statement.